File: java/javadpbo/src/com/sem4dpbo/week2/contohkasus/Kayu.java
package com.sem4dpbo.week2.contohkasus;

public class Kayu {
    private int kodeKayu, berat, volume;
    private String jenisPohon;

    Kayu(){}

    public void setKodeKayu(int kodeKayu) {
        this.kodeKayu = kodeKayu;
    }

    public void setJenisPohon(String jenisPohon) {
        this.jenisPohon = jenisPohon;
    }

    public void setVolume(int volume) {
        this.volume = volume;
    }

    public void setBerat(int berat) {
        this.berat = berat;
    }

    public int getKodeKayu() {
        return kodeKayu;
    }

    public String getJenisPohon() {
        return jenisPohon;
    }

    public int getBerat() {
        return berat;
    }

    public int getVolume() {
        return volume;
    }
}

File: cpp/2nd_Week/ContohKasusKayu/Kayu.cpp
#include <iostream>
#include <string>

using namespace std;

class Kayu
{
    private:
        string kodeKayu;
        string jenisPohon;
        int berat;
        int volume;
    public:
        Kayu(){};

        void setKodeKayu(string kodeKayu){
            this->kodeKayu = kodeKayu;
        }

        void setJenisPohon(string jenisPohon){
            this->jenisPohon = jenisPohon;
        }

        void setBerat(int berat){
            this->berat = berat;
        }
        
        void setVolume(int volume){
            this->volume = volume;
        }

        string getKodeKayu(){
            return this->kodeKayu;
        }

        string getJenisPohon(){
            return this->jenisPohon;
        }

        int getBerat(){
            return this->berat;
        }

        int getVolume(){
            return this->volume;
        }

        ~Kayu(){};
};

